datahub-web-react/src/app/permissions/policy/ManagePolicies.tsx

```diff
@@ -38,7 +38,9 @@ import analytics, { EventType } from '../../analytics';
 import { POLICIES_CREATE_POLICY_ID, POLICIES_INTRO_ID } from '../../onboarding/config/PoliciesOnboardingConfig';
 import { OnboardingTour } from '../../onboarding/OnboardingTour';
 
-const SourceContainer = styled.div``;
+const SourceContainer = styled.div`
+    height: calc(100% - 40px);
```

Comment:
where is 40px coming from?

Reply:
make PageContainer height: calc(100% - 11vh);  SourceContainer height: 100%